index.html

```diff
@@ -352,52 +352,52 @@
-,{"system":"ClickHouse Cloud (gcp)","date":"2025-03-23","machine":"8GiB, 3 replica(s)","cluster_size":"3","comment":"","tags":["C++","column-oriented","ClickHouse derivative","managed","gcp"],"load_time":1009.998,"data_size":9946973751,"result":[[0.002,0.003,0.003],[0.218,0.036,0.023],[0.386,0.176,0.695],[0.841,1.031,0.244],[3.571,2.081,3.626],[6.277,5.882,3.664],[0.151,0.128,0.107],[0.037,0.040,0.039],[5.929,3.005,4.094],[3.079,3.600,3.337],[0.860,0.836,0.866],[1.118,1.172,1.057],[4.067,3.869,3.071],[5.999,4.547,5.954],[6.100,5.353,4.698],[2.681,2.886,2.746],[13.057,10.845,10.630],[7.777,7.460,8.050],[33.473,33.941,30.327],[0.263,0.072,0.072],[5.158,23.666,4.773],[25.230,5.205,5.298],[6.379,23.097,6.137],[61.643,71.516,19.787],[1.587,1.437,1.508],[1.042,1.065,1.016],[1.432,1.506,1.518],[4.860,4.764,4.669],[51.834,51.204,51.287],[0.159,0.166,0.160],[3.312,3.657,3.352],[4.321,4.085,3.489],[30.910,31.034,41.872],[26.196,26.508,27.791],[27.400,27.011,43.649],[1.862,1.797,1.764],[0.176,0.172,0.403],[0.071,0.059,0.061],[0.061,0.073,0.064],[0.333,0.350,0.352],[0.036,0.031,0.033],[0.026,0.021,0.028],[0.026,0.019,0.022]],"source":"clickhouse-cloud/results/gcp.3.8.json"}
+,{"system":"ClickHouse Cloud (aws)","date":"2025-03-25","machine":"12GiB, 1 replica(s)","cluster_size":"1","comment":"","tags":["C++","column-oriented","ClickHouse derivative","managed","aws"],"load_time":468.409,"data_size":9941642893,"result":[[0.002,0.002,0.002],[0.029,0.029,0.029],[0.253,0.270,0.261],[0.506,0.465,0.523],[1.832,1.747,1.552],[3.228,3.175,3.125],[0.058,0.088,0.067],[0.031,0.031,0.031],[2.397,2.321,2.456],[2.362,2.223,2.246],[0.716,0.681,0.730],[0.855,0.926,0.911],[2.291,2.372,2.272],[3.671,4.041,3.865],[3.861,3.672,3.604],[1.916,1.695,1.768],[8.813,8.543,8.739],[6.304,6.634,6.133],[24.453,24.900,26.457],[0.057,0.059,0.059],[4.682,4.540,4.484],[4.821,4.909,4.853],[5.841,5.669,5.575],[16.220,14.951,15.135],[1.347,1.265,1.275],[0.880,0.848,0.858],[1.373,1.302,1.371],[4.399,4.075,4.351],[44.419,39.397,33.459],[0.114,0.109,0.108],[1.844,1.792,1.671],[2.320,2.357,2.140],[16.270,16.697,15.767],[8.712,8.881,8.872],[8.955,9.094,9.039],[0.989,0.980,0.991],[0.088,0.089,0.089],[0.045,0.040,0.040],[0.044,0.043,0.043],[0.194,0.181,0.181],[0.026,0.020,0.022],[0.019,0.016,0.016],[0.017,0.015,0.014]],"source":"clickhouse-cloud/results/aws.1.12.json"}
```

Comment:
CI automatically updates index.html based on the results JSON files in the PR. No need to edit index.html manually.